client/js/README.md

```diff
@@ -1,31 +1,170 @@
-# `@gradio/client`
+# JavaScript Gradio client
 
-A javascript client to call Gradio APIs.
+A javascript (and typescript) client to call Gradio APIs.
 
-**install it**
+## Installation
+
+The Gradio JavaScript client is available on npm as `@gradio/client`. You can install it as below:
 
 ```sh
 pnpm add @gradio/client
```

Comment:
isn't npm much more popular than pnpm? should we lead with that?

Reply:
Yeah good catch, i'll update this.